homeassistant/components/tibber/sensor.py

```diff
@@ -39,7 +53,11 @@ async def async_setup_entry(hass, entry, async_add_entities):
         if home.has_active_subscription:
             dev.append(TibberSensorElPrice(home))
         if home.has_real_time_consumption:
-            dev.append(TibberSensorRT(home))
+            TibberRtDataHandler(async_add_entities, home, hass)
+            hass.bus.async_listen_once(
+                EVENT_HOMEASSISTANT_START,
+                TibberRtDataHandler(async_add_entities, home, hass).hass_started,
```

Comment:
We don't want to create two instances of the `TibberRtDataHandler` I think, right?

Reply:
Yes, we do.
For each home / Tibber pulse device, we will subscribe to the data and add a unique callback function.
(It will be only one open websocket)

Reply:
Fixed

Reply:
Fixed